sdks/swift/Mentat/Mentat/Core/TypedValue.swift

```diff
@@ -0,0 +1,75 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Let's not force our users to conform to both the MPL2.0 and Apache. Switch all of these to Apache.